app/views/comment_mailer/notify_note_author.html.erb

```diff
@@ -19,6 +19,15 @@ Do NOT reply to this email; click this link to respond:
 
 <hr />
 
+<p>Look like spam?
+  <% if @user.user.can_moderate? %>
```

Comment:
Ah.. @qm3 can you please replace this line with - `<% if @user.role == "admin" || @user.role == "moderator" %>` ? Sorry this one took some time. Thanks!

Reply:
@gauravano I committed the new change. This time I didn't create a new merge PR just to see if that works. Please let me know if there is anything I need to do further!